daml-foundations/daml-ghc/daml-prim-src/GHC/Types.daml

```diff
@@ -28,21 +28,22 @@ import GHC.Prim
 
 infixr 5 ::
 
--- | The kind of constraints, like `Show a`
+-- | A type for constraints, like `Show a`
 data Constraint
 
 -- | (Kind) This is the kind of type-level symbols.
 -- Declared here because class IP needs it
 data Symbol
 
--- | Documentation for lists
+-- | A type for lists (`[1,2,3]`)
```

Comment:
Can we omit the `()` brackets round the outside, otherwise it gets confusing, since Haskell does have banana brackets with `([` in some extensions.

Reply:
done